orchestrator/gorc/gorc.toml

```diff
@@ -1,11 +1,14 @@
+keystore = "/tmp/keystore"
+
 [gravity]
 contract = "0x6b175474e89094c44da98b954eedeac495271d0f"
+fees_denom = "stake"
 
 [ethereum]
-key = "testkey"
+key_derivation_path = "m/44'/60'/0'/0/0"
```

Comment:
UX: might be nice to set a default key here

Reply:
We do implement defaults, but aren't exposing it to the user. I think we _should_ do that with something like:
`gorc config show-defaults` (definitely open to better subcommand action names here :))